packages/sitecore-jss/src/dataModels.ts

```diff
@@ -1,3 +1,19 @@
+/**
+ * Phrases from the Sitecore Dictionary Service
```

Comment:
At some point we will start auto-generating an API reference with TypeDoc. Please ensure your TypeScript comments cover function params to (see https://typedoc.org/guides/doccomments/#param-param-name)

Reply:
@anastasiya29 Do you mean to cover only in ./packages/** ?